src/modules/account/components/AddAccountForm/AddAccountForm.js

```diff
@@ -109,27 +108,17 @@ const AddAccountFormContainer = ({
 };
 
 const AddAccountFormWithDerivationPath = (props) => {
-  const [derivationPath, setDerivationPath] = useState(defaultDerivationPath);
-  const derivationPathErrorMessage = useMemo(
-    () => getDerivationPathErrorMessage(derivationPath),
-    [derivationPath]
-  );
-
-  const onDerivationPathChange = (value) => {
-    setDerivationPath(value);
-  };
+  const [derivationPathEvent, setDerivationPathEvent] = useState({
```

Comment:
Why did we do this refactor? With this change we are now using two different variables for identical data. This means that we need to keep track of both to never make sure that they differ, which usually results in buggs and a pain to maintain.

<img width="1046" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/37a9ac2e-c53f-4563-803b-b8f93e2ffe73">
<img width="1165" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/94b747ea-daa3-43d2-8faf-fec14d861109">